### ISSUE TITLE
API class name doesnt match the file name

### ISSUE THREAD

**Issue by gabriel-sportmanager** [ahmadawais/Sendy-PHP-API#3]
Hello, I'm trying to use Sendy-PHP-API in my laravel application, but it can't find the \SENDY\API because of the psr-4... the filename must match with the class name... so it should be changed or to API.php or class Sendy {

let me know if there is any problem in my issue, or if you people solve it by any mean... and sorry for bad english

Gabriel Lemos

**Comment by JorisDebonnet** [comment]
Indeed, I also have this issue (without Laravel)! Composer cannot autoload this package. If you install with composer, there should be no need to use `require_once()` anywhere.

@kaptk2's PR #5 fixes it, but it's a backwards incompatible change for people who currently manually require the file (as requested in the README). Maybe this will warrant a 3.x release if @ahmadawais cares about SemVer :)

**Comment by Fossil01** [comment]
Would be nice if this can finally be fixed. 

```
Deprecation Notice: Class SENDY\API located in ./vendor/ahmadawais/sendy-php-api/src/Sendy.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

@ahmadawais ?

**Comment by JorisDebonnet** [comment]
This is actually fully fixed by #5 now (even backwards compatible). It just needs to be merged in.

**Comment by ahmadawais** [comment]
This was fixed in the v6.0.0. There are breaking changes. You need to now use `\AhmadAwais\Sendy\API` where `API` is the class name and the rest is the namespace.
